src/test/java/io/kestra/plugin/openai/ResponsesTest.java

```diff
@@ -121,6 +121,10 @@ void testFunctionTool() throws Exception {
 
         assertNotNull(output);
         assertNotNull(output.getOutputText());
-        assertThat(output.getOutputText(), containsString("positive"));
+        assertThat(output.getOutputText(), anyOf(
+            containsString("happy"),
+            containsString("enjoy"),
+            containsString("positive")
```

Comment:
I added an `anyOf` because I noticed the output can vary with those adjectives while testing!